app/models/move.rb

```diff
@@ -215,6 +215,10 @@ def important_events
     incident_events + (profile&.person_escort_record&.medical_events || [])
   end
 
+  def vehicle_registration
+    journeys.max_by(&:client_timestamp)&.vehicle&.dig('registration')
```

Comment:
Is there a way to do this at SQL level so that only one matching row is returned? That would be more performant then returning all journeys and iterating through them, if it's possible.

Reply:
Unfortunately the serializer eager loads the journeys before getting to this point, so when we attempt to do this at the SQL level, we get an n+1 query. The performance shouldn't be impacted as there aren't a large number of journeys per move, just a few